src/Fields/Value.php

```diff
@@ -86,7 +86,7 @@ public function antlersValue(Parser $parser, $variables)
         }
 
         if ($this->shouldParseAntlers()) {
-            return $parser->parse($value, $variables);
+            return $this->fieldtype->augment($parser->parse($this->raw(), $variables));
```

Comment:
I think this should be moved above line 82 to the value doesn't need to be retrieved for no reason.